src/store/plugins/tipUrl.js

```diff
@@ -17,6 +17,8 @@ export default (store) =>
         let status;
         if (blacklistedUrls.some((u) => url.includes(u))) {
           status = 'blacklisted';
+        } else if (url.startsWith('http') && !url.startsWith('https:')) {
```

Comment:
What if I'm donating to http.com? Also, let's normalize URL before validation (https://github.com/aeternity/superhero-ui/issues/996)